### PR TITLE
fix(table): remove row documents on delete

### PR DESCRIPTION
**Pull request by fkm**
Hopefully, this will help with the historical API.

### CODE REVIEW

File: addon/components/cf-field/input/table.js [hunk 1]
```diff
@@ -62,6 +63,12 @@ export default Component.extend(ComponentQueryManager, {
 
     yield this.onSave(remainingDocuments);
 
+    // Remove orphaned document.
+    yield this.apollo.mutate({
+      mutation: removeDocumentMutation,
+      variables: { input: { document: document.uuid } }
+    });
```
Comment (anehx) [inline addon/components/cf-field/input/table.js]: I think we need to unload it from the caluma store as well

Comment (fkm) [reply addon/components/cf-field/input/table.js]: Good catch. Done.

**Comment by fkm** [comment]
I hope it will magically set the minus sign on the table row :smiley:

**Pull request merged**